src/shinyapp_example.R

```diff
@@ -48,8 +48,7 @@ ui <- fluidPage(
       textInput("product_id", "Product ID", ""),
       textInput("product_name", "Product Name", ""),
       textInput("keywords", "Keywords", ""),
```

Comment:
I think this line is leftover from having textInput + the HTML. HTML got removed in last commit but not the textInput w/o placeholder text

Reply:
oops!